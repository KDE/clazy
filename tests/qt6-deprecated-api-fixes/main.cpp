#include <QtCore/QTextStream>
#include <QtCore/QCalendar>
#include <QtCore/QDateTime>
#include <QtCore/QDir>
#include <QtCore/QHash>
#include <QtCore/QLinkedList>
#include <QtCore/QMap>
#include <QtCore/QProcess>
#include <QtCore/QResource>
#include <QtCore/QSet>
#include <QtCore/QSignalMapper>
#include <QtCore/QTimeLine>
#include <QtWidgets/QButtonGroup>
#include <QtWidgets/QComboBox>
#include <QtWidgets/QDirModel>
#include <QtWidgets/QDockWidget>
#include <QtWidgets/QGraphicsView>
#include <QtWidgets/QMacCocoaViewContainer>
#include <QtWidgets/QMacNativeWidget>
#include <QtWidgets/QSplashScreen>
#include <QtWidgets/QStyle>
#include <QtWidgets/QTextBrowser>
#include <QtWidgets/QWizard>

class my_Class
{
public:
    QDir m_dir;
    QDir *m_dir_bis;
    QVariant m_variant;
};
#define MYSTRING "myDirPath"

void test()
{
    QDir dir;
    dir = "myStuff";

    QDir d;
    QFileInfo fi;
    d = fi.absolutePath();

    my_Class test_class;
    test_class.m_dir = "name";

    my_Class* test_class_bis = new my_Class;
    test_class_bis->m_dir = ("name");
    *test_class.m_dir_bis = "name";

    QDir dir2;
    dir2 = MYSTRING;
    dir2 = dir;

    QDir dir3;
    dir3= "my" "Stuff";

    QDir dir4;
    char *pathName = "myStuff";
    dir4 = pathName;

    QDir dir5;
    bool cond = true;
    dir5 = cond ? "mystuff" : "yourStuff";

    QDir dir6;
    dir6 = true ? (cond ? "path1" : "path2") : (cond ? "path3" : "path4");

    QDir *dir7 = new QDir("apath");
    *dir7 = "adir";
    ((*dir7)) = "adir";

    QDir::addResourceSearchPath("somePath1");
    dir6.addResourceSearchPath("somePath2");

    QMap<QString, QString> m;
    m.insertMulti("foo", "bar");
    QList<QString> m_keys= m.uniqueKeys();
    QList<QString> m_list= m.values();
    QMap<QString, QString> mm;
    m.unite(mm);

    QProcess pp;
    pp.start("stringContainingACommandWithArguments");
    pp.execute("stringContainingACommandWithArguments");
    pp.startDetached("stringContainingACommandWithArguments");

    QResource rr;
    bool a_bool = rr.isCompressed();

    uint matchtype = 4;
    if (matchtype ==  Qt::MatchRegExp)
        matchtype = 0;

    QTextStream out;
    out << "blabla" << endl;
    out << hex << endl;

    QString a_string = "eeoaaoii";
    QString sep = "o";
    QStringList my_list =  a_string.split(sep, QString::KeepEmptyParts);
    QStringList my_list2 =  a_string.split(sep, QString::SplitBehavior::KeepEmptyParts);
    QString::SplitBehavior behavior = QString::KeepEmptyParts;

    QSet<QString> my_set;
    QSet<QString>::iterator it_set = my_set.begin();
    QSet<QString>::const_iterator cit_set = my_set.cbegin();
    --it_set;
    it_set + 1;
    it_set - 1;
    it_set += 1;
    it_set -= 1;
    ++it_set; //ok
    cit_set -= 2;
    cit_set += 1;
    cit_set + 1;
    cit_set - 1;

    QSetIterator<QString> i(my_set);
    i.hasPrevious();
    i.previous();
    i.peekPrevious();
    i.findPrevious(a_string);

    QSet<int> s;
    s << 1 << 17 << 61 << 127 << 911;
    s.rbegin();
    s.rend();
    s.crbegin();
    s.crend();

    int my_int = 2;
    QWidget* my_qwidget;
    QObject* my_qobject;
    QSignalMapper sigMap;
    sigMap.mapped(1);
    sigMap.mapped(my_int);
    sigMap.mapped("astring");
    sigMap.mapped(a_string);
    sigMap.mapped(my_qwidget);
    sigMap.mapped(my_qobject);

    QHash<QString, QString> my_hash;
    QHash<QString, QString>::iterator it_hash = my_hash.begin();
    QHash<QString, QString>::const_iterator cit_hash = my_hash.cbegin();
    --it_hash;
    it_hash + 1;
    it_hash - 1;
    it_hash += 1;
    it_hash -= 1;
    ++it_hash; //ok
    cit_hash -= 2;
    cit_hash += 1;
    cit_hash + 1;
    cit_hash - 1;

    QHashIterator<QString, QString> ih(my_hash);
    ih.hasPrevious();
    ih.previous();
    ih.peekPrevious();
    ih.findPrevious(a_string);

    QLinkedList<QString> linkList;

    qrand();
    qsrand(1);

    QTimeLine timeline;
    timeline.setCurveShape(QTimeLine::CurveShape::EaseInCurve);
    timeline.curveShape();

    QDate myDate;
    QDate *myOtherDate = new QDate();
    QCalendar myCalendar;
    myDate.toString(Qt::DateFormat::TextDate, myCalendar);
    myDate.toString("format", myCalendar);
    QDateTime myDateTime = QDateTime(myDate);
    myDateTime = QDateTime(*myOtherDate);

    QVariant var1;
    QVariant *var3;
    QVariant var2;
    bool bool1 = var1 > var2;
    bool bool2 = (var1 >= (var2));
    bool bool3 = ((*var3) < var2);
    bool bool4 = (*var3 <= var2);
    bool bool5 = (*var3 <= test_class.m_variant);
    bool bool6 = (test_class_bis->m_variant <= test_class.m_variant);


}

void function1(QLinkedList<QString> arg) {};
QLinkedList<QString> function2() { return {}; };

class aclass
{
public:
     QLinkedList<QString> m_linkList;
     void m_function1(QLinkedList<QString> arg) {};
};

namespace Qt {
    void test_1() {
        uint matchtype = 4;
        if (matchtype ==  MatchRegExp)
            matchtype = 0;
        QTextStream out;
        out << "blabla" << QTextStreamFunctions::endl;
        out << QTextStreamFunctions::hex << QTextStreamFunctions::endl;

        QString a_string = "eeoaaoii";
        QString sep = "o";
        QStringList my_list =  a_string.split(sep, QString::KeepEmptyParts);
        QString::SplitBehavior behavior = QString::KeepEmptyParts;

    }
}

void test_widgets()
{
    QWizard wizard;
    wizard.visitedPages();

    QButtonGroup buttonGroup;
    buttonGroup.buttonClicked(1);
    buttonGroup.buttonPressed(1);
    buttonGroup.buttonReleased(1);
    buttonGroup.buttonToggled(1, true);

    QComboBox combobox;
    combobox.setSizeAdjustPolicy(QComboBox::SizeAdjustPolicy::AdjustToMinimumContentsLength);
    QString astring = "asdf";
    combobox.currentIndexChanged(astring);
    combobox.activated(astring);
    combobox.highlighted(astring);

    QMacCocoaViewContainer *cocoa = NULL;
    QMacNativeWidget *native = NULL;

    QWidget *a_widget = NULL;
    QSplashScreen *splash1 = new QSplashScreen(a_widget);
    QScreen *a_screen = NULL;
    QSplashScreen *splash2 = new QSplashScreen(a_screen);

    QTextBrowser browser;
    QString a_string = "aaa";
    browser.highlighted(a_string);
    QUrl a_url;
    browser.highlighted(a_url);

    QDockWidget dock;
    dock.setFeatures(QDockWidget::AllDockWidgetFeatures);

    QDirModel dirModel;

    QGraphicsView graphview;
    QMatrix matrix = graphview.matrix();
    graphview.setMatrix(matrix);
    graphview.resetMatrix();

    QStyle *style = NULL;
    style->pixelMetric(QStyle::PixelMetric::PM_DefaultTopLevelMargin);
    style->pixelMetric(QStyle::PixelMetric::PM_DefaultChildMargin);
    style->pixelMetric(QStyle::PixelMetric::PM_DefaultLayoutSpacing);
    style->subElementRect(QStyle::SubElement::SE_DialogButtonBoxLayoutItem, NULL);

}

void func_coco (QMacCocoaViewContainer *cocoa) {}
void func_native (QMacNativeWidget *native){}
