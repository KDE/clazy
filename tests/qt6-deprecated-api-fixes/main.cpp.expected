qt6-deprecated-api-fixes/main.cpp:58:22: warning: ISO C++11 does not allow conversion from string literal to 'char *' [-Wwritable-strings]
qt6-deprecated-api-fixes/main.cpp:37:9: warning:  function setPath() has to be used in Qt6 [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:41:7: warning:  function setPath() has to be used in Qt6 [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:44:22: warning:  function setPath() has to be used in Qt6 [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:47:27: warning:  function setPath() has to be used in Qt6 [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:48:27: warning:  function setPath() has to be used in Qt6 [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:51:10: warning:  function setPath() has to be used in Qt6 [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:55:9: warning:  function setPath() has to be used in Qt6 [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:59:10: warning:  function setPath() has to be used in Qt6 [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:63:10: warning:  function setPath() has to be used in Qt6 [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:66:10: warning:  function setPath() has to be used in Qt6 [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:69:11: warning:  function setPath() has to be used in Qt6 [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:70:15: warning:  function setPath() has to be used in Qt6 [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:72:5: warning: call function QDir::addResourceSearchPath(). Use function QDir::addSearchPath() with prefix instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:73:10: warning: call function QDir::addResourceSearchPath(). Use function QDir::addSearchPath() with prefix instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:76:7: warning: Use QMultiMap for maps storing multiple values with the same key. [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:77:30: warning: Use QMultiMap for maps storing multiple values with the same key. [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:78:30: warning: Use QMultiMap for maps storing multiple values with the same key. [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:80:7: warning: Use QMultiMap for maps storing multiple values with the same key. [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:83:8: warning: call function QProcess::start(). Use function QProcess::startCommand() instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:88:22: warning: call function QRessource::isCompressed(). Use function QProcess::compressionAlgorithm() instead. [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:91:23: warning: call Qt::MatchRegExp. Use Qt::MatchRegularExpression instead. [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:95:24: warning: call function QTextStream::endl. Use function Qt::endl instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:96:12: warning: call function QTextStream::hex. Use function Qt::hex instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:96:19: warning: call function QTextStream::endl. Use function Qt::endl instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:100:48: warning: Use Qt::SplitBehavior variant instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:101:49: warning: Use Qt::SplitBehavior variant instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:102:5: warning: Using QString::SplitBehavior. Use Qt::SplitBehavior variant instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:102:39: warning: Use Qt::SplitBehavior variant instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:107:5: warning: QSet iterator categories changed from bidirectional to forward. Please port your code manually [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:108:12: warning: QSet iterator categories changed from bidirectional to forward. Please port your code manually [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:109:12: warning: QSet iterator categories changed from bidirectional to forward. Please port your code manually [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:110:12: warning: QSet iterator categories changed from bidirectional to forward. Please port your code manually [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:111:12: warning: QSet iterator categories changed from bidirectional to forward. Please port your code manually [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:113:13: warning: QSet iterator categories changed from bidirectional to forward. Please port your code manually [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:114:13: warning: QSet iterator categories changed from bidirectional to forward. Please port your code manually [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:115:13: warning: QSet iterator categories changed from bidirectional to forward. Please port your code manually [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:116:13: warning: QSet iterator categories changed from bidirectional to forward. Please port your code manually [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:119:7: warning: QSet iterator categories changed from bidirectional to forward. Please port your code manually [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:120:7: warning: QSet iterator categories changed from bidirectional to forward. Please port your code manually [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:121:7: warning: QSet iterator categories changed from bidirectional to forward. Please port your code manually [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:122:7: warning: QSet iterator categories changed from bidirectional to forward. Please port your code manually [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:126:7: warning: QSet iterator categories changed from bidirectional to forward. Please port your code manually [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:127:7: warning: QSet iterator categories changed from bidirectional to forward. Please port your code manually [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:128:7: warning: QSet iterator categories changed from bidirectional to forward. Please port your code manually [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:129:7: warning: QSet iterator categories changed from bidirectional to forward. Please port your code manually [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:135:12: warning: call function QSignalMapper::mapped(int). Use function QSignalMapper::mappedInt(int) instead. [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:136:12: warning: call function QSignalMapper::mapped(int). Use function QSignalMapper::mappedInt(int) instead. [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:137:12: warning: call function QSignalMapper::mapped(const QString &). Use function QSignalMapper::mappedString(const QString &) instead. [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:138:12: warning: call function QSignalMapper::mapped(const QString &). Use function QSignalMapper::mappedString(const QString &) instead. [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:139:12: warning: call function QSignalMapper::mapped(QWidget *). Use function QSignalMapper::mappedObject(QWidget *) instead. [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:140:12: warning: call function QSignalMapper::mapped(QObject *). Use function QSignalMapper::mappedObject(QObject *) instead. [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:145:5: warning: QHash iterator categories changed from bidirectional to forward. Please port your code manually [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:146:13: warning: QHash iterator categories changed from bidirectional to forward. Please port your code manually [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:147:13: warning: QHash iterator categories changed from bidirectional to forward. Please port your code manually [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:148:13: warning: QHash iterator categories changed from bidirectional to forward. Please port your code manually [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:149:13: warning: QHash iterator categories changed from bidirectional to forward. Please port your code manually [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:151:14: warning: QHash iterator categories changed from bidirectional to forward. Please port your code manually [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:152:14: warning: QHash iterator categories changed from bidirectional to forward. Please port your code manually [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:153:14: warning: QHash iterator categories changed from bidirectional to forward. Please port your code manually [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:154:14: warning: QHash iterator categories changed from bidirectional to forward. Please port your code manually [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:157:8: warning: QHash iterator categories changed from bidirectional to forward. Please port your code manually [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:158:8: warning: QHash iterator categories changed from bidirectional to forward. Please port your code manually [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:159:8: warning: QHash iterator categories changed from bidirectional to forward. Please port your code manually [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:160:8: warning: QHash iterator categories changed from bidirectional to forward. Please port your code manually [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:162:5: warning: Using QLinkedList. Use std::list instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:164:5: warning: use QRandomGenerator instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:165:5: warning: use QRandomGenerator instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:168:14: warning: call QTimeLine::setCurveShape. Use QTimeLine::setEasingCurve instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:169:14: warning: call QTimeLine::curveShape. Use QTimeLine::easingCurve instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:174:47: warning: replacing with function omitting the calendar. Change manually and use QLocale if you want to keep the calendar. [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:176:28: warning: deprecated constructor. Use QDate::startOfDay() instead. [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:177:18: warning: deprecated constructor. Use QDate::startOfDay() instead. [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:182:23: warning:  operator does not exist in Qt6. Using QVariant::compare() instead. [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:183:24: warning:  operator does not exist in Qt6. Using QVariant::compare() instead. [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:184:27: warning:  operator does not exist in Qt6. Using QVariant::compare() instead. [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:185:25: warning:  operator does not exist in Qt6. Using QVariant::compare() instead. [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:186:25: warning:  operator does not exist in Qt6. Using QVariant::compare() instead. [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:187:45: warning:  operator does not exist in Qt6. Using QVariant::compare() instead. [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:192:16: warning: Using QLinkedList. Use std::list instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:193:1: warning: Using QLinkedList. Use std::list instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:198:6: warning: Using QLinkedList. Use std::list instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:199:23: warning: Using QLinkedList. Use std::list instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:205:27: warning: call Qt::MatchRegExp. Use Qt::MatchRegularExpression instead. [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:208:28: warning: call function QTextStream::endl. Use function Qt::endl instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:209:16: warning: call function QTextStream::hex. Use function Qt::hex instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:209:45: warning: call function QTextStream::endl. Use function Qt::endl instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:213:52: warning: Use Qt::SplitBehavior variant instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:214:9: warning: Using QString::SplitBehavior. Use Qt::SplitBehavior variant instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:214:43: warning: Use Qt::SplitBehavior variant instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:222:12: warning: call function QProcess::visitedPages(). Use function QProcess::visitedIds() instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:225:17: warning: call function QButtonGroup::buttonClicked(int). Use function QButtonGroupidClicked instead. [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:226:17: warning: call function QButtonGroup::buttonPressed(int). Use function QButtonGroupidPressed instead. [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:227:17: warning: call function QButtonGroup::buttonReleased(int). Use function QButtonGroupidReleased instead. [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:228:17: warning: call function QButtonGroup::buttonToggled(int, bool). Use function QButtonGroupidToggled instead. [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:231:34: warning: Use QComboBox::SizeAdjustPolicy::AdjustToContents or AdjustToContentsOnFirstShow instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:233:14: warning: Use currentIndexChanged(int) instead, and get the text using itemText(index). [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:234:14: warning: Using QComboBox::activated(const QString &). Use textActiated() instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:235:14: warning: Using QComboBox::hilighted(const QString &). Use textHighlighted() instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:237:5: warning: Using QMacCocoaViewContainer. Use QWindow::fromWinId and QWidget::createWindowContainer instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:238:5: warning: Using QMacNativeWidget. Use QWidget::winId instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:241:34: warning: Use the constructor taking a QScreen * instead. [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:247:13: warning: Using QTextBrowser::highlighted(const QString &). Use QTextBrowser::highlighted(const QUrl &) instead. [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:252:22: warning: Use QComboBox::DockWidgetClosable|DockWidgetMovable|DockWidgetFloatable explicitly instead. [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:254:5: warning: Using QDirModel. Use QFileSystemModel instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:257:32: warning: Using QGraphicsView::matrix. Use transform() instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:258:15: warning: Using QGraphicsView::setMatrix(const QMatrix &). Use setTransform(const QTransform &) instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:259:15: warning: Using QGraphicsView::resetMatrix(). Use resetTransform() instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:262:24: warning: this enum has been removed in Qt6 [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:263:24: warning: this enum has been removed in Qt6 [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:264:24: warning: this enum has been removed in Qt6 [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:265:27: warning: this enum has been removed in Qt6 [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:269:17: warning: Using QMacCocoaViewContainer. Use QWindow::fromWinId and QWidget::createWindowContainer instead [-Wclazy-qt6-deprecated-api-fixes]
qt6-deprecated-api-fixes/main.cpp:270:19: warning: Using QMacNativeWidget. Use QWidget::winId instead [-Wclazy-qt6-deprecated-api-fixes]
